packages/test/src/activities/default-and-defined.ts

```diff
@@ -0,0 +1,13 @@
+export interface NameAndArgs {
+  name: string;
+  activityName?: string;
+  args: any[];
+}
+
+export async function definedActivity(...args: unknown[]): Promise<NameAndArgs> {
+  return { name: 'definedActivity', args };
+}
+
+export default async function (...args: unknown[]): Promise<NameAndArgs> {
+  return { name: 'default', activityName: 'definedActivity', args };
```

Comment:
Why is this hardcoded? Seems like a bug to me. Look into `activityInfo()`

```
activityName: 'definedActivity'
```

Reply:
Indeed a bug (or at least not representative of what we're actually trying to test).
I suspect it was some lazy copy pasting on my part. Fixed